server/src/controllers/AuthController.js

```diff
@@ -47,6 +49,21 @@ module.exports = {
 
       const user = await User.create(req.body);
 
+      if (process.env.NODE_ENV === "production")
+      {
+        var defaultClient = SibApiV3Sdk.ApiClient.instance;
+        var apiKey = defaultClient.authentications['api-key'];
+        apiKey.apiKey = process.env.SIB_API_KEY;
+        var apiInstance = new SibApiV3Sdk.ContactsApi();
```

Comment:
Can we move the apiInstance into config ? That way if we require some other functionalities, we can pass the apiInstance across.

Reply:
Sure will do.